components/game/game.js

```diff
@@ -1,25 +1,65 @@
 import gameStyles from './game.module.scss';
-import { useState } from 'react';
+import styles from "@/app/page.module.css";
+import { useState, useEffect } from 'react';
 import Hand from "@/components/gameComponents/hand.js";
-import { dictionaryCombinations } from '@/components/utilities/combination.js';
+import { dictionaryCombinations, highestValue } from '@/components/utilities/combination.js';
+import { mapCard } from '../utilities/card';
+
+const icons = {
+  'hearts': '♥',
+  'diamonds': '♦',
+  'spades': '♠',
+  'clubs': '♣',
+}
 
 const Game = () => {
+  const [shuffledDeck, setDeck] = useState([]);
   const [deckIsShuffled, shuffleDeck] = useState(false);
   const [introIsVisible, showIntro] = useState(false);
   const [playerTurn, setPlayerTurn] = useState(0);
   const [hands, setHands] = useState(null);
   const [comboIsValid, setComboStatus] = useState(null);
   const [currentTurnCombo, setCurrentTurnCombo] = useState('single');
+  const [previousPlayedCombo, setPreviousPlayedCombo] = useState([]);
 
   const [selectCombo, setComboSelect] = useState('single');
-
   // Build Card Deck
   const suites = ['spades', 'clubs', 'diamonds', 'hearts'];
   // 3-10 are normal cards, 11 is Jack, 12 is Queen, 13 is King, 14 is Ace, 15 is 2.
-  const numbers = ['3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15'];
+  const numbers = [3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
   let value = 1; // 3 of spades is the lowest card
   const deck = numbers.map((number, idx) => suites.map((suite, i) => ({ number, suite, value: value++ }))).flat();
 
+  useEffect(() => {
+    setDeck(shuffle(deck));
+  }, []);
+
+  useEffect(() => {
+     /**
+     * @description Prompt ai logic
+     * NOTE: This is logic for AI players
+     */
+
+    if (playerTurn !== 0) {
+      let valueToBeat;
+      if (previousPlayedCombo.length === 0) {
+        valueToBeat = 0;
+      } else {
+        valueToBeat = previousPlayedCombo[previousPlayedCombo.length - 1].value;
+      }
+
+      const currHand = hands[playerTurn].hand;
+      const lowestCard = currHand.reduce((lowest, curr) => {
+        if(curr.value < lowest && curr.value > valueToBeat) {
+          return curr.value;
+        }
+        return lowest;
+      }, 53);
+      console.log(`Player ${playerTurn + 1} plays: ${currHand.find((card) => card.value === lowestCard).number} of ${currHand.find((card) => card.value === lowestCard).suite}`);
+      requestCombo([currHand.find((card) => card.value === lowestCard)], 'single');
```

Comment:
note to dan: if CPU doesn’t have applicable card to play, it breaks.

Reply:
alright couple things here. i would make use of variables here. something as simple as this would help with readability:

```javascript
const cardToPlay = currHand.find((card) => card.value === lowestCard);
if(cardToPlay) {
  requestCombo([cardToPlay], 'single');
} else {
  console.log(`Player ${playerTurn + 1} passes.`);
  passTurn();
}
```

^^ just note that by passing here,right now, without ending the "cycle" this will just cause an endless loop of passing.